release.sh

```diff
@@ -71,7 +71,14 @@ then
   exit 1
 fi
 
-echo Copying over base Dockerfiles
+printf "\n\nDeleting old container images\n\n"
+
+until docker images | grep "containers.schibsted.io/finntech/node" | awk "{print $3}" | sort | uniq | xargs docker rmi -f
```

Comment:
and finally you can replace the loop construct with xargs' `-L1` like this to run it for every line:

`➜  ~ docker images | grep containers.schibsted.io/finntech/node | awk '{print $3}' | sort | uniq | xargs -L1 docker rmi -f`